spring-cloud-app-broker-docs/src/docs/asciidoc/service-instances.adoc

```diff
@@ -127,16 +129,199 @@ The following table lists properties that can be set for all or specific app dep
 
 ===== Environment Configuration
 
+You can provide environment variables to be set on a deployed app. Environment variables are set using properties under `environment` for the deployed app, as in the following example:
+
+[subs=+quotes]
+```yml
+spring:
+  cloud:
+    appbroker:
+      services:
+        - service-name: example
+          plan-name: standard
+          apps:
+            - name: example-service-app1
+              path: classpath:app1.jar
+              *environment:*
+                *logging.level.spring.security: DEBUG*
+                *spring.profiles.active: cloud*
+```
+
 ===== Service Configuration
 
+You can configure services which should be bound to a deployed app. Services are configured using properties under `services` for the deployed app, as in the following example:
+
+[subs=+quotes]
+```yml
+spring:
+  cloud:
+    appbroker:
+      services:
+        - service-name: example
+          plan-name: standard
+          apps:
+            - name: example-service-app1
+              path: classpath:app1.jar
+              *services:*
+              *- service-instance-name: example-db*
+                *name: mysql*
+                *plan: small*
+                *parameters:*
+                  *param-key: param-value*
+```
+
 ==== Dynamic Customization
 
 To customize the backing application deployment using information that is only available when performing a service broker operation or that must be generated per service instance, you can use the service broker application configuration to provide the names of customization implementations.
 
 ===== Backing Application Target
 
+You can configure the target location for backing applications (in Cloud Foundry, an org and space) using a `target` specification, as in the following example:
+
+[subs=+quotes]
+```yml
+spring:
+  cloud:
+    appbroker:
+      services:
+        - service-name: example
+          plan-name: standard
+          *target:*
+          *- name: SpacePerServiceInstance*
+        apps:
+          apps:
+            - name: example-service-app1
+              path: classpath:app1.jar
+```
+
+By default (if you do not provide a `target` specification), all backing applications are deployed to the default target specified under `spring.cloud.appbroker.deployer`. For Cloud Foundry, this is the org named by `spring.cloud.appbroker.deployer.cloudfoundry.default-org` and the space named by `spring.cloud.appbroker.deployer.cloudfoundry.default-space`.
+
+====== The `SpacePerServiceInstance` Target
```

Comment:
The second `Target` option is `ServiceInstanceGuidSuffix`, as shown in this issue: https://github.com/spring-cloud/spring-cloud-app-broker/issues/78